src/features/inventory/index.js

```diff
@@ -33,6 +33,24 @@ class Inventory extends Component {
     }
   }
 
+  handleKeyDown(event) {
+    event.preventDefault();
+    switch(event.keyCode) {
+      case 13:
+      case 32:
+        // Don't toggle the inventory visibility with any of these keys
+        // because right now, they're being used for the attack order.
+        break;
+      case 73:
```

Comment:
Since we are only concerned with a single key here, we could maybe avoid using a `switch` altogether. Doing something like
```
if (event.keyCode === 73) {
    event.stopPropagation();
    this._toggleInventory();
}
```
would probably work just as well.